files/helpers.py

```diff
@@ -366,18 +369,31 @@ def media_file_info(input_file):
         stdout = run_command(cmd).get("out")
         stream_size = sum([int(line) for line in stdout.split("\n") if line != ""])
         video_bitrate = round((stream_size * 8 / 1024.0) / video_duration, 2)
+    
+    if "r_frame_rate" in video_info.keys():   
```

Comment:
can you output contents of `video_info["r_frame_rate"]` here? could this break now, or on future ffmpeg versions, so would it be better to ask it more defensively? After all these are not used anywhere (at least on this file!)

Reply:
I probably do not understand what you mean. Would you like it like this?
```
if video_info["r_frame_rate"]:
        video_frame_rate = video_info["r_frame_rate"].partition("/")
        video_frame_rate_n =  video_frame_rate[0]
        video_frame_rate_d =  video_frame_rate[2]  
```
It is used here:
```
target_fps = Fraction(int(media_info.get("video_frame_rate_n", 30)), int(media_info.get("video_frame_rate_d", 1)))
```
and there are default values for the case the if statement comes our false.

Reply:
I was just worried whether there is a chance that video_frame_rate[0] or video_frame_rate[2] will result in IndexError!

Reply:
It is tuple. I tried to put nonsense in video_info["r_frame_rate"] but the worst case is that it does nothing. No IndexError.